content/source/docs/enterprise/private/capacity.html.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Capacity and Performance"
+sidebar_current: "docs-enterprise2-private-installer-capacity"
+---
+
+# Private Terraform Enterprise Capacity and Performance (Installer)
+
+The maximum capacity and performance of Private Terraform Enterprise (PTFE) is dependent entirely on the resources
+provided by the Linux instance it is installed on. There are a few settings that allow the capacity that PTFE
+uses to be adjusted to suit the instance.
+
+## Memory + Concurrency
+
+The amount of memory to allocate to a Terraform run and the number of concurrent runs are the primary elements in
+understanding capacity above the base services.
+
+By default, PTFE allocates 256 MB of memory to each Terraform run, with a default concurrency of 10 parallel runs.
+Therefore, by default PTFE requires 2.6 GB of memory reserved for runs.
+
+After factoring in the memory needed to run the base services that make up the application, the default memory footprint of PTFE is approximately 4 GB.
+
+### Settings
+
+The settings for per-run memory and concurrency are available in the dashboard on port 8800, on the Settings page, under the Capacity section. They can also be set via
+the [application settings JSON file when using the automated install procedure](https://www.terraform.io/docs/enterprise/private/automating-the-installer.html#available-settings).
```

Comment:
@evanphx This is probably the one substantive change I made, since this was referring to the replicated.conf but I think it's in the app settings. Can you confirm?

Reply:
Oh yer. Yes! Thank you.